docs/introduction/framework-extensibility.md

```diff
@@ -7,6 +7,7 @@ as well as a list of customizations that are not (and will not be) possible at a
 ## What is achievable easily
 * [Extending an entity](/docs/wip_glassbox/entity-extension.md)
     * [Adding a new attribute](/docs/cookbook/adding-new-attribute-to-an-entity.md)
+    * *Beware the [OrderItem has some limitations](/docs/wip_glassbox/entity-extension.md#orderitem)*
```

Comment:
I would replace beware with note like this:
`*Note: There are some limitations when extending OrderItem, for more ....*